pkg/raft/scripts/fix.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
Do we need this `pkg/raft/scripts` package?

Reply:
No, removed.